### PR TITLE
add Solace Crafting

### LINKED ISSUE #1893: [Egg Request]: <Solace Crafting>
### Does this expand an already existing service

No

### Link to a game or software

https://solacecrafting.com

### Links for the download

https://store.steampowered.com/app/670260/Solace_Crafting/

### Links for the install docs

https://solace-crafting.fandom.com/wiki/Multiplayer_Hosting_Guide

### PR DESCRIPTION
**Pull request by gOOvER**
# Description

Add egg for Solace Crafting. STEAM AUTH NEEDED
Fixes #1893 

## Checklist for all submissions

* [x] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [x] Have you tested and reviewed your changes with confidence that everything works?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

## New egg Submissions

1. [x] Does your submission pass tests (server is connectable)?
2. [ ] Does your egg use a custom docker image?
    * [ ] Have you tried to use a generic image?
    * [ ] Did you PR the necessary changes to make it work?
3. [x] Have you added the egg to the main README.md and any other README files in subdirectories of the egg (e.g /game_eggs) according to the alphabetical order?
4. [x] Have you added a unique README.md for the egg you are adding according to the alphabetical order?

### CODE REVIEW

**Review by BrickBotTM — CHANGES_REQUESTED** [review]
The app id should be 1086950 so it can be logging in as anonymous for install. 1086950 is for the normal dedicated server. That way you don't need someone who owns the game to install the server. So you don't need the variables.

**Comment by gOOvER** [comment]
> The app id should be 1086950 so it can be logging in as anonymous for install. 1086950 is for the normal dedicated server. That way you don't need someone who owns the game to install the server. So you don't need the variables.

both appids are the same. The dev never switched to anonymous login ;) Read the discord; i asked now 1000 times and the dev told me 1000 times he changed it

**Comment by gOOvER** [comment]
> The app id should be 1086950 so it can be logging in as anonymous for install. 1086950 is for the normal dedicated server. That way you don't need someone who owns the game to install the server. So you don't need the variables.

ty for the tip. The dev was not able to give me this info. I ixx the egg AppId

**Comment by gOOvER** [comment]
Updated egg and remove the complete login

**Review by BrickBotTM — APPROVED** [review]
Works like a charm now!

File: game_eggs/steamcmd_servers/solace_crafting/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+### Server Ports
+
+| Port  | default |
+|-------|---------|
+| Game  | 27015   |
+| Query | 27016   |
+
+### SteamGuard
+
+To start this Server, Steam Login is needed
```
Comment (QuintenQVD0) [inline game_eggs/steamcmd_servers/solace_crafting/README.md:14]: So this is then not more needed in the readme ?

**Review by QuintenQVD0 — CHANGES_REQUESTED** [review]

**Review by QuintenQVD0 — APPROVED** [review]

**Pull request merged**